src/DblParser/dune

```diff
@@ -1,5 +1,10 @@
-(ocamlyacc (modules YaccParser))
 (ocamllex  (modules Lexer))
+;(ocamlyacc (modules YaccParser));
+
+(rule
+ (deps YaccParser.mly)
+ (target YaccParser.ml)
```

Comment:
Does cpspg generate `.mli` files? If yes, maybe it should be added to this rule (I'm sure, dune can do that. You can ask @Foxinio, our local dune guru). If not, maybe we could consider writing own `YaccParser.mli` file.

Reply:
It does not generate `.mli` files, but it should be straightforward to implement.